batch/processor_Notifications_Token.py

```diff
@@ -95,19 +99,22 @@ def _gen_block_timestamp(entry):
     @staticmethod
     def _get_token_all_list(db_session: Session):
         _tokens = []
-        listed_tokens: list[Listing] = db_session.query(Listing).all()
-        for listed_token in listed_tokens:
-            if not token_list.is_registered(listed_token.token_address):
-                continue
-            else:
-                token_type = token_list.get_token(listed_token.token_address)[1]
-                _tokens.append({
-                    "token": listed_token,
-                    "token_type": token_type
-                })
+        registered_tokens: list[IDXTokenListItem] = (
+            db_session.query(IDXTokenListItem).
+            join(Listing, and_(Listing.token_address == IDXTokenListItem.token_address)).all()
+        )
+        for registered_token in registered_tokens:
+            _tokens.append({
+                "token": registered_token,
+                "token_type": registered_token.token_template
+            })
         return _tokens
 
-    def db_merge(self, db_session: Session, token_contract, token_type, log_entries):
+    def db_merge(
+        self, db_session: Session,
+        token_contract: Web3Contract, token_type: str,
+        log_entries: list[EventData], token_owner_address: str
+    ):
         pass
```

Comment:
Add `token_owner_address` to arguments so as to remove `Contract.call_function` in `db_merge` method.